src/patternfly/components/Drawer/docs/code.md

```diff
@@ -0,0 +1,11 @@
+## Usage
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-drawer` | `<div>` | Creates the drawer container. |
+| `.pf-c-drawer__main` | `<div>` | Creates the drawer main container. |
```

Comment:
Is the reasoning behind calling this `pf-c-drawer__main` because content from the page's main section goes inside of it? I feel like main implies that its the main thing which I'm not sure it is, what about `pf-c-drawer__content`?